content/docs/latest/user/streaming-data-ingest-v2.md

```diff
@@ -79,15 +79,15 @@ HiveStreamingConnection API also supports 2 partitioning mode (static vs dynamic
 
 See the [Javadoc for HiveStreamingConnection](http://hive.apache.org/javadocs/r3.0.0/api/org/apache/hive/streaming/HiveStreamingConnection.html) for more information. 
 
 #### Usage Guidelines
 
 Generally, the more records are included in each transaction the more throughput can be achieved.  It's common to commit either after a certain number of records or after a certain time interval, whichever comes first.  The later ensures that when event flow rate is variable, transactions don't stay open too long.  There is no practical limit on how much data can be included in a single transaction. The only concern is amount of data which will need to be replayed if the transaction fails. The concept of a TransactionBatch serves to reduce the number of files (and delta directories) created by HiveStreamingConnection API in the filesystem. Since all transactions in a given transaction batch write to the same physical file (per bucket), a partition can only be compacted up to the the level of the earliest transaction of any batch which contains an open transaction.  Thus TransactionBatches should not be made excessively large.  It makes sense to include a timer to close a TransactionBatch (even if it has unused transactions) after some amount of time.
 
-The HiveStreamingConnection is highly optimized for write throughput ([Delta Streaming Optimizations](http://hive.apache.org/javadocs/r3.0.0/api/org/apache/hive/streaming/HiveStreamingConnection.Builder.html#withStreamingOptimizations-boolean-)) and as a result the delta files generated by Hive streaming ingest have many of the ORC features disabled (dictionary encoding, indexes, compression, etc.) to facilitate high throughput writes. When the compactor kicks in, these delta files get rewritten into read- and storage-optimized ORC format (enable dictionary encoding, indexes and compression). So it is recommended to configure the compactor more aggressively/frequently (refer to [Compactor](https://cwiki.apache.org/confluence/display/Hive/Hive+Transactions#HiveTransactions-Compactor)) to generate compacted and optimized ORC files.
+The HiveStreamingConnection is highly optimized for write throughput ([Delta Streaming Optimizations](http://hive.apache.org/javadocs/r3.0.0/api/org/apache/hive/streaming/HiveStreamingConnection.Builder.html#withStreamingOptimizations-boolean-)) and as a result the delta files generated by Hive streaming ingest have many of the ORC features disabled (dictionary encoding, indexes, compression, etc.) to facilitate high throughput writes. When the compactor kicks in, these delta files get rewritten into read- and storage-optimized ORC format (enable dictionary encoding, indexes and compression). So it is recommended to configure the compactor more aggressively/frequently (refer to [Compactor](https://hive.apache.org/docs/latest/user/hive-transactions#compactor)) to generate compacted and optimized ORC files.
```

Comment:
Outside scope of PR but the URL doesn't work: http://hive.apache.org/javadocs/r3.0.0/api/org/apache/hive/streaming/HiveStreamingConnection.Builder.html#withStreamingOptimizations-boolean-